examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldClient.java

```diff
@@ -49,23 +49,25 @@
   private final PriorityBlockingQueue<Long> latencies = new PriorityBlockingQueue<>();
   private final AtomicInteger failedRpcs = new AtomicInteger();
 
+  protected Map<String, ?> getHedgingServiceConfig() {
+    return new Gson()
+            .fromJson(
+                    new JsonReader(
+                            new InputStreamReader(
+                                    HedgingHelloWorldClient.class.getResourceAsStream(
+                                            "hedging_service_config.json"),
+                                    UTF_8)),
+                    Map.class);
+  }
+
   /** Construct client connecting to HelloWorld server at {@code host:port}. */
   public HedgingHelloWorldClient(String host, int port, boolean hedging) {
-    Map<String, ?> hedgingServiceConfig =
-      new Gson()
-          .fromJson(
-              new JsonReader(
-                  new InputStreamReader(
-                      HedgingHelloWorldClient.class.getResourceAsStream(
-                          "hedging_service_config.json"),
-                      UTF_8)),
-              Map.class);
-
     ManagedChannelBuilder<?> channelBuilder = ManagedChannelBuilder.forAddress(host, port)
         // Channels are secure by default (via SSL/TLS). For the example we disable TLS to avoid
         // needing certificates.
         .usePlaintext();
     if (hedging) {
+      Map<String, ?> hedgingServiceConfig = getHedgingServiceConfig();
```

Comment:
Move the Map creation to this line; don't add the getHedgingServiceConfig method. And please use 4 spaces instead of 8 for indentation (we actually use 2, but 4 for line continuations; after the first indent of 4 spaces, the rest could have been 2 spaces).

The method should have been a private method and adds an indirection. We want our examples to be very concrete and clear, which means favoring less indirection and more "it looks like a basic script."